app/main.py

```diff
@@ -2,24 +2,66 @@
 
 
 class IntegerRange:
```

Comment:
It is a good practice to provide a docstring for the class to explain its purpose and how it should be used.